spec/duration.html

```diff
@@ -1438,34 +1438,33 @@ <h1>
+          1. Let _dateDuration2_ be ! CreateTemporalDuration(_duration2_.[[Years]], _duration2_.[[Months]], _duration2_.[[Weeks]], _duration2_.[[Days]], 0, 0, 0, 0, 0, 0).
           1. Let _dateAdd_ be ? GetMethod(_calendar_, *"dateAdd"*).
           1. Let _intermediate_ be ? CalendarDateAdd(_calendar_, _relativeTo_, _dateDuration1_, *undefined*, _dateAdd_).
           1. Let _end_ be ? CalendarDateAdd(_calendar_, _intermediate_, _dateDuration2_, *undefined*, _dateAdd_).
           1. Let _dateLargestUnit_ be ! LargerOfTwoTemporalUnits(*"day"*, _largestUnit_).
           1. Let _differenceOptions_ be OrdinaryObjectCreate(*null*).
           1. Perform ! CreateDataPropertyOrThrow(_differenceOptions_, *"largestUnit"*, _dateLargestUnit_).
           1. Let _dateDifference_ be ? CalendarDateUntil(_calendar_, _relativeTo_, _end_, _differenceOptions_).
-          1. Let _result_ be ? BalanceDuration(_dateDifference_.[[Days]], _h1_ + _h2_, _min1_ + _min2_, _s1_ + _s2_, _ms1_ + _ms2_, _mus1_ + _mus2_, _ns1_ + _ns2_, _largestUnit_).
+          1. Let _result_ be ? BalanceDuration(_dateDifference_.[[Days]], _duration1_.[[Hours]] + _duration2_.[[Hours]], _duration1_.[[Minutes]] + _duration2_.[[Minutes]], _duration1_.[[Seconds]] + _duration2_.[[Seconds]], _duration1_.[[Milliseconds]] + _duration2_.[[Milliseconds]], _duration1_.[[Microseconds]] + _duration2_.[[Microseconds]], _duration1_.[[Nanoseconds]] + _duration2_.[[Nanoseconds]], _largestUnit_).
           1. Return ! CreateDurationRecord(_dateDifference_.[[Years]], _dateDifference_.[[Months]], _dateDifference_.[[Weeks]], _result_.[[Days]], _result_.[[Hours]], _result_.[[Minutes]], _result_.[[Seconds]], _result_.[[Milliseconds]], _result_.[[Microseconds]], _result_.[[Nanoseconds]]).
         1. Assert: _relativeTo_ has an [[InitializedTemporalZonedDateTime]] internal slot.
         1. Let _timeZone_ be _relativeTo_.[[TimeZone]].
         1. Let _calendar_ be _relativeTo_.[[Calendar]].
-        1. Let _intermediateNs_ be ? AddZonedDateTime(_relativeTo_.[[Nanoseconds]], _timeZone_, _calendar_, _y1_, _mon1_, _w1_, _d1_, _h1_, _min1_, _s1_, _ms1_, _mus1_, _ns1_).
-        1. Let _endNs_ be ? AddZonedDateTime(_intermediateNs_, _timeZone_, _calendar_, _y2_, _mon2_, _w2_, _d2_, _h2_, _min2_, _s2_, _ms2_, _mus2_, _ns2_).
+        1. Let _endNs_ be ? AddZonedDateTime(_relativeTo_, _timeZone_, _calendar_, _duration1_.[[Years]] + _duration2_.[[Years]], _duration1_.[[Months]] + _duration2_.[[Months]], _duration1_.[[Weeks]] + _duration2_.[[Weeks]], _duration1_.[[Days]] + _duration2_.[[Days]], _duration1_.[[Hours]] + _duration2_.[[Hours]], _duration1_.[[Minutes]] + _duration2_.[[Minutes]], _duration1_.[[Seconds]] + _duration2_.[[Seconds]], _duration1_.[[Milliseconds]] + _duration2_.[[Milliseconds]], _duration1_.[[Microseconds]] + _duration2_.[[Microseconds]], _duration1_.[[Nanoseconds]] + _duration2_.[[Nanoseconds]]).
```

Comment:
This change is normative, because it replaces two `AddZonedDateTime` calls with a single call to `AddZonedDateTime`. Apart from the obvious normative changes (e.g. fewer observable lookups), will this change result in any other changes, i.e. was there a reason to perform two calls?

Reply:
Nice catch! I have reverted the normative change and referenced this comment from #2289.

As for whether or not the separate calls make sense, I don't think so. They seem harmless but unnecessary for Temporal.Duration.prototype.{add,subtract}, and undesirable for Temporal.Duration.prototype.round and Temporal.ZonedDateTime.prototype.{since,until}.